### PR TITLE
Create anubhav823.json

### PR DESCRIPTION
**Pull request by anubhav823**

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]
have some record pls?

**Comment by anubhav823** [comment]
I have added a record.

File: domains/anubhav823.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "anubhav823' personal developer website",
+  "repo": "",
+  "owner": {
+    "username": "anubhav823",
+    "email": "anubhav823@gmail.com"
+  },
+  "record": {
+    "CNAME": "anubhav823.github.io"
```
Comment (MaskDuck) [inline domains/anubhav823.json:9]: 404?

**Review by MaskDuck — DISMISSED** [review]

File: domains/anubhav823.json (hunk 1 above)
Comment (anubhav823) [reply domains/anubhav823.json]: I have updated the record

**Review by mtgsquad — DISMISSED** [review]

File: domains/anubhav823.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "anubhav823' personal developer website",
+  "repo": "",
+  "owner": {
+    "username": "anubhav823",
+    "email": "anubhav823@gmail.com"
+  },
+  "record": {
+    "CNAME": "anubhav823.github.io/wordle"
```
Comment (mtgsquad) [inline domains/anubhav823.json:9]: You cannot use paths in CNAME records, if you'd like to redirect to `anubhav823.github.io/wordle` then please use a URL record.

**Review by mtgsquad — CHANGES_REQUESTED** [review]

File: domains/anubhav823.json (hunk 2 above)
Comment (anubhav823) [reply domains/anubhav823.json]: I have changed the record type.

**Review by mtgsquad — CHANGES_REQUESTED** [review]
Please add the protocol before `anubhav823.github.io/wordle`, and make it: `https://anubhav823.github.io/wordle`

**Comment by anubhav823** [comment]
Updated the url by adding the protocol

**Review by mtgsquad — APPROVED** [review]

**Pull request merged**